modules/order/order.graphql.js

```diff
@@ -29,55 +39,55 @@ enum Status {
   SENT
   DELIVERED
 }
-type OrderItems {
-  category: [Language]
-  subcategory: [Language]
-  model: [Language]
-  name: [Language]
-  colors: [[Language]]
-  pattern: [Language]
-  closure: [Language]
-  closureColor: String
-  size: Size,
-  bottomMaterial: [Language]
-  bottomColor: [Language]
-  additions: [[Language]]
-  actualPrice: [CurrencySet]
+type OrderItem {
+  product: Product
+  model: Model
   quantity: Int
+  isFromConstructor: Boolean
+  options: ItemOptions
+  constructorBasic: ConstructorBasic
+  constructorPattern: Pattern
+  constructorFrontPocket: ConstructorFrontPocket
+  constructorBottom: ConstructorBottom
+  fixedPrice: [CurrencySet]
 }
 type Delivery {
   sentOn: String
-  sentBy: String
+  sentBy: SendByEnum
   invoiceNumber: String
   courierOffice: Int
   byCourier: Boolean
   cost: [CurrencySet]
 }
+enum SendByEnum{
+  NOVAPOST
+  UKRPOST
+  SELFPICKUP
+}
 type OrderUser {
   firstName: String
   lastName: String
   patronymicName: String
   email: String
   phoneNumber: String
 }
+type ItemOptions{
+  size: Size
+  sidePocket: Boolean
+}
 `;
 const orderInputs = ` 
 input OrderInput {
-  _id:ID
   status: Status
   user: OrderUserInput,
   delivery: DeliveryInput,
-  items: [OrderItemsInput],
+  items: [OrderItemInput],
   paymentMethod: PaymentEnum
   userComment: String
-  adminComment: String
-  cancellationReason: String
-  address: AddressInput
-  dateOfCreation: Date
-  lastUpdatedDate: Date
   totalItemsPrice: [CurrencyInputSet]
   totalPriceToPay: [CurrencyInputSet]
```

Comment:
We don't need this inputs
It will be better to calculate total price on back-end based on items price, delivery and etc.

Reply:
fixed